docs/standard/base-types/parsing-datetime.md

```diff
@@ -19,57 +16,81 @@ helpviewer_keywords:
+  
+The format provider is also used to interpret an ambiguous numeric date. It is not clear which components of the date represented by the string "02/03/04" are the month, day, and year. The components are interpreted according to the order of similar date formats in the format provider.
+
+## Parse
+
+The following example illustrates the use of the <xref:System.DateTime.Parse%2A?displayProperty=nameWithType> method to convert a `string` into a <xref:System.DateTime>. This example uses the culture associated with the current thread. If the <xref:System.Globalization.CultureInfo> associated with the current culture cannot parse the input string, a <xref:System.FormatException> is thrown.
+
+> [!TIP]
+> All the C# samples in this article run in your browser. Press the **Run** button to see the output. You can also edit them to experiment yourself.
+
+> [!NOTE]
+> These examples are available in the GitHub docs repo for both [C#](https://github.com/dotnet/docs/samples/tree/master/snippets/csharp/how-to/conversions) and [VB](https://github.com/dotnet/docs/samples/tree/master/snippets/visualbasic/how-to/conversions). Or, you can download the project as a zipfile for [C#]((https://github.com/dotnet/docs/samples/tree/master/snippets/csharp/how-to/conversions.zip) or [VB](https://github.com/dotnet/docs/samples/tree/master/snippets/visualbasic/how-to/conversions.zip).
+
+[!code-csharp-interactive[Parsing.DateAndTime#1](../../../samples/snippets/csharp/how-to/conversions/StringToDateTime.cs#1)]
+[!code-vb[Parsing.DateAndTime#1](../../../samples/snippets/visualbasic/how-to/conversions/Program.vb#1)]
+
+You can also request  a <xref:System.Globalization.CultureInfo> when you parse a string. You specify one of the standard <xref:System.Globalization.DateTimeFormatInfo> objects returned by the <xref:System.Globalization.CultureInfo.DateTimeFormat%2A?displayProperty=nameWithType> property. The following example uses a format provider to parse a German string into a <xref:System.DateTime>. It creates a <xref:System.Globalization.CultureInfo> representing the `de-DE` culture. That `CultureInfo` object ensures successful parsing of this particular string. This precludes whatever setting is in the <xref:System.Threading.Thread.CurrentCulture> of the <xref:System.Threading.Thread.CurrentThread>.  
```

Comment:
There's a space between "request" and "a", but also request is the wrong verb, since the caller supplies it (or it's supplied implicitly) rather than requests it. But rather than define this in terms of the object involved, it could also be rephrased in terms of what the method call accomplishes: "You can also explicitly define the culture whose formatting conventions are used when you parse a string."

Reply:
Good suggestion.